melos.yaml

```diff
@@ -33,7 +33,7 @@ scripts:
 
   prepare:demos:
     description: Download SQLite3 wasm for demos
-    run: dart run powersync:setup_web --no-worker
+    run: dart run powersync_core:setup_web --no-worker
```

Comment:
Should this rather stay in the `powersync` package?

1. Existing applications may depend on having `dart run powersync:setup_web` available.
2. The downloaded WASM files are specific to plain SQLite, and powersync_sqlcipher (if we ever support that on web) would need to use different files.

Reply:
My original thought was that we could reuse it for SQLCipher but it isn't web compatible at this point so I moved it back into the `powersync` package.